main.go

```diff
@@ -371,18 +371,21 @@ func main() {
 		}()
 	}
 
-	var registerClient = k8s.AdmissionWebhookRegisterClient{
-		Logger: logger.Named("admissionWebhookRegisterClient"),
-	}
+	// Deploy mutating webhook manifest
+	if conf.RegisterWebhookClient == "true" {
+		var registerClient = k8s.AdmissionWebhookRegisterClient{
+			Logger: logger.Named("admissionWebhookRegisterClient"),
+		}
 
-	err = registerClient.Register(ctx, conf)
-	if err != nil {
-		prod.Fatal(err.Error())
-	}
+		err = registerClient.Register(ctx, conf)
```

Comment:
Plz fix indentation here. I guess the Tab key was used instead of the Space key. Please run "go fmt".

Reply:
Not sure why in the UI it is showing like this but i am following correct indentation rules:
![Screenshot from 2024-01-09 13-13-59](https://github.com/kubeslice/cmd-admission-webhook-k8s/assets/57394571/322bde01-71a2-4550-8161-018f84b79367)

- [x] I have also ran `go fmt ./...`